hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1700,6 +1700,45 @@
       .</description>
   </property>
 
+  <property>
+    <name>ozone.om.ratis.snapshot.dir</name>
+    <value/>
+    <tag>OZONE, OM, STORAGE, MANAGEMENT, RATIS</tag>
+    <description>This directory is used for storing OM's snapshot
+      related files like the ratisSnapshotIndex and DB checkpoint from leader
+      OM.
+      If undefined, OM snapshot dir will fallback to ozone.om.ratis.storage.dir.
```

Comment:
Minor: In code if ozone.om.ratis.snapshot.dir not defined, next fallback is om.ratis.storage.dir, if this too is not set it fall backs to ozone.om.ratis.storage.dir.

Reply:
The fallback is in this order:
ozone.om.ratis.snapshot.dir -> ozone.om.ratis.storage.dir -> ozone.metadata.dir

Reply:
Ya ur right. In my earlier comment, the last thing I mean is ozone.metadata.dir, but written it wrongly.
Ya, it looks good to me. As we already have fall back mentioned for ozone.om.ratis.storage.dir during the property description.